### DISCUSSION TITLE
Permission Management or Parental Controls

### DISCUSSION THREAD

**Discussion by Jdiesel87** [advplyr/audiobookshelf#404] [Ideas]
I'd not sure how to best phase this but what are everyone's thoughts on how to best manage various users ability to access content on a common library?

It is a bit tough with something like books where there is no standard age rating or specific cases where you want to limit access to specific books on a case per case basis. The solution right now would be to create a second library containing the books you want to limit access to but this could be considered cumbersome especially if your have more than a few users.

What would a practical solution be? Being able to tag books and limit individual users to certain tags or untagged books? A simple toggle to mark a book as being only visible to a certain user class?

**Comment by Jdiesel87** [comment]
Relevant FR
https://github.com/advplyr/audiobookshelf/issues/402

**Reply by advplyr** [reply]
A lot of the services like iTunes/Audible API's come with an Explicit flag. I can add this flag onto the new data model then we can add a user setting to hide explicit content.

**Reply by Jdiesel87** [reply]
That sounds like a perfectly reasonable approach

**Reply by snowmangoh** [reply]
An additional (would be my preferred) option could be limiting a user to a collection that has been created (are those cross-user?). I do that with Plex, as not all rating are always accurate, and not all PG-13s are the same.

**Reply by snowmangoh** [reply]
I like the tag idea as well.
